modules/index.js

```diff
@@ -44,7 +44,15 @@ const PLUGINS = {
   },
   // Martech
   martech: { url: `${pluginBasePath}/martech.js`, when: ({ urlParameters }) => urlParameters.size > 0 },
-  onetrust: { url: `${pluginBasePath}/onetrust.js`, when: () => (document.querySelector('#onetrust-consent-sdk') || hasCookieKey('OptanonAlertBoxClosed')), isBlockDependent: true },
+  onetrust: {
+    url: `${pluginBasePath}/onetrust.js`,
+    when: () => (hasCookieKey('OptanonAlertBoxClosed')) || document.querySelector('#onetrust-consent-sdk'),
```

Comment:
first parenthesis is not useful, either you put the 2 parts of the OR condition in parenthesis or you 
```suggestion
    when: () => (hasCookieKey('OptanonAlertBoxClosed') || document.querySelector('#onetrust-consent-sdk')),
```